app/models/user.rb

```diff
@@ -2,5 +2,25 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+  :recoverable, :rememberable, :trackable, :validatable, :omniauthable, omniauth_providers: [:facebook]
```

Comment:
I would put devise :omniauthable, omniauth_providers: [:facebook] on a seperate line to improve readability and clearness of what it does.